src/models/salt.js

```diff
@@ -0,0 +1,33 @@
+import crypto from 'crypto'
+
+export default (sequelize, DataTypes) => {
+  const Salt = sequelize.define('salt', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    value: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+  })
+  // Class methods
+  Salt.getPasswordSalt = async function () {
+    const lastSalt = await this.getLast()
+
+    if (lastSalt) return lastSalt
+
+    return this.createRandom()
```

Comment:
We can join lines 19 and 21 in a single line doing `return lastSalt || this.createRandom()`

Reply:
Done !